environment.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - numpy
   - scipy
   - pyyaml
-  - pyccl>=2.5.0,<2.8.0
+  - pyccl>=2.3.0
```

Comment:
@carlosggarcia , I guess we should fix this dependence at 3.0.0, since we don't expect this to work for pyccl <=2.7.0

Reply:
right, that was my intention. my bad